lib/raidenclient/RaidenClient.ts

```diff
@@ -83,22 +83,24 @@ class RaidenClient extends BaseClient {
 
     this.port = port;
     this.host = host;
+
     if (disable) {
-      this.setStatus(ClientStatus.DISABLED);
+      return;
     }
+    this.setStatus(ClientStatus.DISCONNETED);
+
+    this.init();
   }
 
   /**
    * Discover our raiden address.
    */
   public async init() {
-    if (!this.isDisabled()) {
```

Comment:
I think we should leave this disabled check in place, we don't want to allow initialization if Raiden is supposed to be disabled.

Reply:
done